test/unit/EntityTest.js

```diff
@@ -70,3 +71,16 @@ test('canPlaceBlockOver', t => {
   t.end();
 });
 
+test('sheep Drop', t => {
+  const sheep = new Sheep(mockGameController, "Sheep", 1, 1, "Sheep", true, 1);
+
+  // Sheep starts !naked
+  t.false(sheep.naked);
```

Comment:
I wonder if we should change this property to `sheared` instead.

Reply:
What if a sheep is naked because it just doesn't grow fur (hair? wool? Is it still called wool, pre-shearing?) and not because it was sheared!?

Either makes sense to me, if anyone some day is passionate about it, it could be changed with little effort.